descriptors/ModuleDescriptor-template.json

```diff
@@ -220,6 +220,19 @@
             "circulation-storage.loans.item.delete"
           ]
         },
+        {
+          "methods": [
+            "PUT"
+          ],
+          "pathPattern": "/circulation/loans/{id}/declare-lost",
+          "permissionsRequired": [
+            "circulation.loans.item.declare.lost"
+          ],
+          "modulePermissions": [
```

Comment:
There also needs to be the permissions needed to fetch all of the records needed, e.g. the loan, item, holdings, instance, service points, timezone etc

Reply:
Also, in order to avoid the permissions becoming too large, endpoint permissions tend to be separated into a permission set.

Reply:
Why do you think that the permission to fetch all loan, item, holdings, instance, service points, timezone etc is needed here?

Reply:
I haven’t analysed it in much detail, I imagine the loan and item are fetched before they are updated, and so all the relevant permissions for doing that are needed

Reply:
All the permissions required are already there in modperms.circulation.loans.item.put

Reply:
Hmm, is that reusing the permission set for the PUT endpoint for a loan?

Reply:
I think we shouldn't be reusing other permission sets for operations unless they are intended to be identical in the actions they perform and even then it's probably not a good idea as very few things remain identical for long. Each endpoint/method should have all its dependencies well defined and include only those permissions that satisfy the dependencies. We've already found and closed several security loopholes with respect to permissions. I think we need to be careful with what we assign and always use the principle of least privilege.

Reply:
I agree with @mreno-EBSCO 

If that means copying and pasting the set for the moment, I think that’s what we should do.

Last quarter, I had thoughts about great info fine grained internal permission sets for heavily related activities. Whilst that would likely lead to some undue escalation, it might be a worthwhile compromise for easing the maintenance overhead. I haven’t had chance to experiment and determine if that is in any way viable.

Reply:
Done.